packages/forma-36-react-components/src/components/Tag/Tag.stories.tsx

```diff
@@ -1,58 +1,69 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, select } from '@storybook/addon-knobs';
 
-import Tag, { tagType } from './Tag';
+import Tag, { TagProps } from './Tag';
 import SectionHeading from '../Typography/SectionHeading';
 import Flex from '../Flex/Flex';
 
-storiesOf('Components/Tag', module)
-  .addParameters({
-    propTypes: Tag['__docgenInfo'],
-    component: Tag,
-  })
-  .add('default', () => (
-    <Tag
-      tagType={select(
-        'tagType',
-        {
-          'Primary (default)': 'primary',
-          Positive: 'positive',
-          Negative: 'negative',
-          Warning: 'warning',
-          Secondary: 'secondary',
-          Muted: 'muted',
-        },
-        'primary',
-      )}
-      entityStatusType={select(
-        'Entity Status Type',
-        {
-          Published: 'published',
-          Archived: 'archived',
-          draft: 'draft',
-          changed: 'changed',
-          deleted: 'deleted',
-          'None (default)': undefined,
-        },
-        undefined,
-      )}
-      className={text('className', '')}
-    >
-      {text('Children', 'Published')}
-    </Tag>
-  ))
-  .add('overview', () => (
-    <>
-      <Flex marginBottom="spacingS">
-        <SectionHeading element="h3">Tag types overview</SectionHeading>
+export default {
+  title: 'Components/Tag',
+  component: Tag,
+  parameters: {
+    propTypes: [Tag['__docgenInfo']],
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+  },
+};
+
+export const basic = (args: TagProps) => (
+  <Tag tagType={args.tagType}>{args['Tag Text']}</Tag>
+);
+
+basic.args = {
+  tagType: 'primary',
+  'Tag Text': 'Published',
+};
```

Comment:
Oh I missed this

I think you could make it simpler like this:

```jsx
export const Basic = ({ tagText, ...args}: TagProps) => (
  <Tag {...args}>{tagText}</Tag>
);

Basic.args = {
   tagType: 'primary',
   tagText: 'Published',
};
```

Reply:
sure, make sense. I will fix it.

Reply:
@gui-santos ✅